lib/chef/client.rb

```diff
@@ -285,6 +286,7 @@ def run
         run_status.start_clock
         logger.info("Starting #{ChefUtils::Dist::Infra::PRODUCT} Run for #{node.name}")
         run_started
+        Chef::Telemetry.run_starting({})
```

Comment:
Should adding telemetry be part of this PR? It feels like that should be separate, this PR, AFAICT, is to fix tests on main where kitchen tests are getting license errors.

Reply:
This is not needed in this PR and removed it.